SA3/qa_web/settings.py

```diff
@@ -129,3 +130,12 @@
 
 # User identification and custom profile
 AUTH_USER_MODEL = 'qa_web.User'
+
+import os
+HAYSTACK_CONNECTIONS = {
+    'default': {
+        'ENGINE': 'haystack.backends.whoosh_backend.WhooshEngine',
+        'PATH': os.path.join(os.path.dirname(__file__), 'whoosh_index'),
+    },
+}
+# HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
```

Comment:
If this is not necessary, maybe we could delete it?

Reply:
It is a important option. I will explain it in quickstart.

Reply:
After testing it out, I believe this line should be uncommented and the setting used in the project. Having to manually update the index is definitely not what we want and this setting allows for dynamic updating of the index.